### PR TITLE
Adaptive captcha-solver

### PR DESCRIPTION
**Pull request by QIN2DIM**
### Background

Provide a ubiquitous human-machine challenge module to ensure that crawlers can access deep information on the Internet in a concurrent state.

The related kits run based on the Selenium or Playwright. We can launch a headless browser that escapes human-machine validation, or beat the challenge at the source level (sync / async mode).

### Changes

For now, all the code is done only in `autogpt/commands/captchas`.

### Documentation

The following is the expected functionality

- [ ] Image to Text
- [x] [hCaptcha [general]](https://accounts.hcaptcha.com/demo)
- [x] [reCaptcha v2 [audio/img]](https://www.google.com/recaptcha/api2/demo)
- [ ] [reCaptcha v3](https://recaptcha-demo.appspot.com/)
- [ ] [Turnstile](https://www.cloudflare.com/zh-cn/products/turnstile/)
- [x] [GeeTest](https://www.geetest.com/show)
- [ ] [163dun](https://dun.163.com/trial/inference)
- [ ] [FunCaptcha](https://api.funcaptcha.com/tile-game-lite-mode/fc/api/nojs/?pkey=69A21A01-CC7B-B9C6-0F9A-E7FA06677FFC&lang=zh)
- [ ] [Epic TalonService](https://talon-website-prod.ak.epicgames.com/)

### PR Quality Checklist
- [x] My pull request is atomic and focuses on a single change.
- [ ] I have thoroughly tested my changes with multiple different prompts.
- [ ] I have considered potential risks and mitigations for my changes.
- [ ] I have documented my changes clearly and comprehensively.
- [x] I have not snuck in any "extra" small tweaks changes

### CODE REVIEW

**Comment by hdkiller** [comment]
This should be a draft.

**Comment by QIN2DIM** [comment]
IN PROGRESS🤖(/▽＼)

**Comment by QIN2DIM** [comment]
FINE

 It looks like a big project. 🥲

**Review by nponeccop — APPROVED** [review]

**Review by nponeccop — APPROVED** [review]

**Comment by kinance** [comment]
Related to #1944

**Comment by ntindle** [comment]
This is awesome! Let’s get some test and it up to date!

**Comment by QIN2DIM** [comment]
## TL;DR
These are some of the intended feats that I've made into separate projects, and I'm trying to merge them gracefully into the AutoGPT 0.0

# hcap-area
<div align="center"></div>

<div align="center">



</div>



# hcap-gen

<div>

</div>


## Slider
<div align="center">



https://user-images.githubusercontent.com/62018067/165540682-9321ebee-a02f-4988-8616-b14e8acb5113.mp4
</div>

<div align="center"></div>

<div align="center"></div>

## recap-audio
<div align="center">

</div>

**Comment by mirikking** [comment]
Wow, very cool progress, I wonder if he can bypass the talon service from epic.

**Comment by Pocker88** [comment]
Hello Dear developer, how about ReCaptcha img solver?!? Or this only for audio method?!?

**Comment by p-i-** [comment]
This is a mass message from the AutoGPT core team.
Our apologies for the ongoing delay in processing PRs.
This is because we are re-architecting the AutoGPT core!

For more details (and for infor on joining our Discord), please refer to:
  https://github.com/Significant-Gravitas/Auto-GPT/wiki/Architecting

**Comment by ntindle** [comment]
This would be amazing if updated to the new system
